README.md

```diff
@@ -1 +1,11 @@
-# Hello-Microverse
\ No newline at end of file
+# Hello-Microverse
+
+# Authors - Amen-Musingarimi
+
+# client - html and css
+
+# Server - local server
+
+# Database - local
+
+# Key-Feature - home-page
```

Comment:
- [ ] As requested by the previous reviewer, your README file does no still look prefessional yet. Check this [repo](https://github.com/Hassaanjbaig-code/Awersome-books) and learn how to customise README template to fit your project.